db/db_impl.cc

```diff
@@ -101,6 +102,7 @@
 
 namespace rocksdb {
 const std::string kDefaultColumnFamilyName("default");
+const std::string kPersistentStatsColumnFamilyName("_stats_history");
```

Comment:
Can we make it even less likely to cause a conflict? For example, put "rocksdb" as a part of the name, and having more "_" than just a leading one?